exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -511,6 +511,11 @@ private ExecConstants() {
       new OptionDescription("Available as of Drill 1.17. Sets the implicit column name for the lastModifiedTime column. " +
           "For internal usage when producing Metastore analyze."));
 
+  public static final String IMPLICIT_PROJECT_METADATA_COLUMN_LABEL = "drill.exec.storage.implicit.project_metadata.column.label";
+  public static final OptionValidator IMPLICIT_PROJECT_METADATA_COLUMN_LABEL_VALIDATOR = new StringValidator(IMPLICIT_PROJECT_METADATA_COLUMN_LABEL,
+      new OptionDescription("Available as of Drill 1.18. Sets the implicit column name for the $project_metadata$ column. " +
```

Comment:
The "Available as of Drill 1.18." is not needed here, only in the docs. If the user is using an earlier version, they won't even see this option.